allennlp/service/predictors/sentence_tagger.py

```diff
@@ -0,0 +1,44 @@
+from overrides import overrides
+
+from allennlp.common.util import JsonDict, sanitize
+from allennlp.data import DatasetReader, Instance
+from allennlp.data.tokenizers.word_splitter import SpacyWordSplitter
+from allennlp.models import Model
+from allennlp.service.predictors.predictor import Predictor
+
+
+@Predictor.register('sentence-tagger')
+class SentenceTaggerPredictor(Predictor):
+    """
+    Wrapper for any model that takes in a sentence and returns
+    a single set of tags for it.  In particular, it can be used with
+    the :class:`~allennlp.models.crf_tagger.CrfTagger` model
+    and also
+    the :class:`~allennlp.models.simple_tagger.SimpleTagger` model.
+    """
+    def __init__(self, model: Model, dataset_reader: DatasetReader) -> None:
+        super().__init__(model, dataset_reader)
+        self._tokenizer = SpacyWordSplitter(language='en_core_web_sm', pos_tags=True)
+
+    @overrides
+    def _json_to_instance(self, json: JsonDict) -> Instance:
```

Comment:
Not necessarily for this PR, just thinking about how we can make the API better here, so we don't have to override `predict_json` all the time.  What if `_json_to_instance` returned a `Tuple[Instance, JsonDict]`, where the json dict contains additional things to pass to the output?

Reply:
I agree we should fix this problem, let me think a little bit about what's the best approach